tests/integration/repo_commits_test.go

```diff
@@ -51,6 +53,18 @@ func doTestRepoCommitWithStatus(t *testing.T, state string, classes ...string) {
 	// Call API to add status for commit
 	t.Run("CreateStatus", doAPICreateCommitStatus(NewAPITestContext(t, "user2", "repo1"), path.Base(commitURL), api.CommitStatusState(state)))
 
+	runBody := doAPICreateCommitStatus(NewAPITestContext(t, "user2", "repo1"), path.Base(commitURL), api.CommitStatusState(state))
```

Comment:
```suggestion
	// Call API to add status for commit, even in parallel
	runBody := doAPICreateCommitStatus(NewAPITestContext(t, "user2", "repo1"), path.Base(commitURL), api.CommitStatusState(state))
	t.Run("CreateStatus", runBody)
	
```
Should hopefully not change any behavior, unless we depend on the test context being unique…